Workbooks/Azure Monitor - Agents/AMA Health/AMA Health -Preview.workbook

```diff
@@ -0,0 +1,1986 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# AMA Health Dashboard -preview"
```

Comment:
if you include this `-preview` in the *name of the file* later you'll want to rename the file to not be preview, and then in version control in git it will show up as a rename/move and will be harder to look at history.  i'd suggest just naming it what you want it to be, and only put the "preview" in the gallery title/etc.

Reply:
Good idea! I'll rename the workbook without the "preview."